lib/driver.js

```diff
@@ -33,6 +31,13 @@ class WebDriverAgentDriver extends BaseDriver {
     return {};
   }
 
+  async getNativeDriver () {
```

Comment:
If we want to add this method, we should use it everywhere. Meaning all uses of `this.wda` should get updated.

Curious why we want this here though.